pinc/charsuite-symbols-collection.inc

```diff
@@ -2,7 +2,7 @@
 include_once($relPath."CharSuites.inc");
 
 $charsuite = new CharSuite("symbols-collection", _("Symbols collection"));
-$charsuite->description = _("Astronomical, zodiac, and a limited number of apothecary and music symbols. The zodiac symbols are provided with the variation selector, VS15, to try to force them to display as ordinary charcters instead of as emoji. This may not work for all fonts.");
+$charsuite->description = _("Astronomical, zodiac, and a limited number of apothecary and music symbols. The zodiac symbols are provided with the variation selector, VS15, to try to force them to display as ordinary characters instead of as emoji. This may not work for all fonts.");
```

Comment:
While we're here did we want to do the update to this recommended by bunny-crunch?

Reply:
Good idea, thanks. I would like to verify with him my understanding of what he thinks it should say, so probably tomorrow before I get the change pushed.

Reply:
Done.